btree/btree.c

```diff
@@ -1927,7 +1919,8 @@ static void node_put(struct node_op *op, struct nd *node, struct m0_be_tx *tx)
 			shift = node->n_type->nt_shift(node);
 			m0_free(node);
 			m0_rwlock_write_unlock(&list_lock);
-			segops->so_node_free(op, shift, tx, 0);
+			m0_free_aligned(segaddr_addr(&op->no_addr),
```

Comment:
We need to capture this operation in transaction. Also `m0_free_aligned` is used at two places.  Put `m0_free_aligned` in another function and call this function from the two places. for eg. 
```
seg_node_free(op, shift, tx)
{
m0_free_aligned(segaddr_addr(&op->no_addr),1ULL << shift, shift);
/** Capture in transaction */
}

Reply:
Sure. I will update it.